__tests__/createBrowserLikeFetch.spec.js

```diff
@@ -46,6 +46,29 @@ describe('createCookiePassingFetch', () => {
     expect(setCookie.mock.calls[0][2].maxAge).toEqual(3600000);
   });
 
+  it('does not add missing propterties', async () => {
```

Comment:
```suggestion
  it('does not add missing or null properties', async () => {
```

Reply:
the null properties get introduced by tough cookie so are an internal detail

Reply:
```suggestion
  it('does not add missing properties', async () => {
```